bagit.xml

```diff
@@ -581,19 +584,24 @@ As a result, no <spanx style="emph">filename</spanx> listed in a tag manifest be
               octets (8-bit bytes) across all payload file content and
               <spanx style="emph">StreamCount</spanx> is the total number of
               payload files.
+              This metadata element &must-not; be repeated.
           </t>
           <t hangText="Bag-Group-Identifier">
               A sender-supplied identifier for the set, if any, of bags
               to which it logically belongs.
               This identifier must be unique across the sender's content, and if
               recognizable as belonging to a globally unique scheme, the receiver
               should make an effort to honor reference to it.
+              This metadata element &should-not; be repeated.
           </t>
           <t hangText="Bag-Count">
               Two numbers separated by "of", in particular, "N of T",
               where T is the total number of bags in a group of bags and N is the
               ordinal number within the group; if T is not known, specify it as "?"
               (question mark).  Examples:  1 of 2, 4 of 4, 3 of ?, 89 of 145.
+              This metadata element &should-not; be repeated.
+              If this metadata element is present, the Bag-Group-Identifier element
```

Comment:
I don't see the justification for tying this to Bag-Group-Identifier. I'd recommend leaving this less specified.

Reply:
I think the justification is if you have multiple bags (which is implied with `Bag-Count`) then you should have sort of identifier. I think it is good practice but perhaps beyond the scope of the specification.

Reply:
Perhaps `SHOULD` is too much here and instead say something about best practices?

Reply:
Changed to RECOMMENDED in e45f0f1